RevenueCatUI/CustomerCenter/Data/CustomerCenterActionHandler.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import RevenueCat
+
+/// Allows to be notified of certain events the customer may perform during the Customer Center flow
+public protocol CustomerCenterActionHandler {
```

Comment:
I've been wondering whether to rename this to `CustomerCenterActionDelegate` instead, since it might be more common in iOS... Wdyt?

Reply:
Yes, I think that should be a `Delegate`. But maybe @aboedo can seal the deal since he has more iOS experience

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/4057/commits/da03b7c1b275eb29081bd312dde155194f36e5af

Reply:
Hmm actually swiftlint is complaining with the new name: `warning: Class Delegate Protocol Violation: Delegate protocols should be class-only so they can be weakly referenced (class_delegate_protocol)`. I guess this means we should mark it as `@objc` which would make it so we can't implement it on structs like this... I'm wondering whether to just revert the rename or follow the convention specified by swiftlint...